modules/swagger-codegen/src/main/resources/scala/apiInvoker.mustache

```diff
@@ -52,6 +52,9 @@ class ApiInvoker(val mapper: ObjectMapper = ScalaJsonUtil.getJsonMapper,
   def escape(value: String): String = {
     URLEncoder.encode(value, "utf-8").replaceAll("\\+", "%20")
   }
+  def escape(values: List[String]): String = {
```

Comment:
Later we will need to add another method `def escape(values: List[String], format: String): String = { ... }` to support other collection formats.

cc @clasnake